engine/runtime-compiler/src/main/scala/org/enso/compiler/context/ModuleContext.scala

```diff
@@ -25,6 +25,7 @@ case class ModuleContext(
   isGeneratingDocs: Boolean                    = false,
   pkgRepo: Option[PackageRepository]           = None
 ) {
+  def getCompilerModule()      = module
```

Comment:
Adding this method lets `CompilerContext.Module` escape. There is no point in encapsulating access to module's attributes with `def isSynthetic()`, `def bindingsAnalysis()`, etc.

Why is the access to `CompilerContext.Module` needed now?

Reply:
Removed this method once again in 0d38ad2ed4e728379c60016ac06d926a063752b3 - I had to do some refactorings for that.